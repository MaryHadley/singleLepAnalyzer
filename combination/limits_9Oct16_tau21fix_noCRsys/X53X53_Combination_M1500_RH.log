000 (1297363.000.000) 10/10 19:27:08 Job submitted from host: <192.168.63.249:41533>
...
001 (1297363.000.000) 10/10 19:27:18 Job executing on host: <192.168.63.230:40000>
...
006 (1297363.000.000) 10/10 19:27:26 Image size of job updated: 63504
	10  -  MemoryUsage of job (MB)
	9948  -  ResidentSetSize of job (KB)
...
006 (1297363.000.000) 10/10 19:32:26 Image size of job updated: 282960
	161  -  MemoryUsage of job (MB)
	164080  -  ResidentSetSize of job (KB)
...
006 (1297363.000.000) 10/10 20:12:32 Image size of job updated: 284028
	163  -  MemoryUsage of job (MB)
	166200  -  ResidentSetSize of job (KB)
...
005 (1297363.000.000) 10/10 20:17:14 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:49:19, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:49:19, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1633  -  Run Bytes Sent By Job
	323  -  Run Bytes Received By Job
	1633  -  Total Bytes Sent By Job
	323  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :     1097        1   3824539
	   Memory (MB)          :      163     3072      3072
...
