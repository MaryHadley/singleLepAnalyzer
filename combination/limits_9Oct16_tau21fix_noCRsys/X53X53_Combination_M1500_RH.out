/var/lib/condor/execute/dir_16103
Total number of likelihood evaluations: 55000101
Total number of likelihood evaluations: 5500101
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1500_RH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1500_RH/quant-toys:0-sig-5f68156f77.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1500_RH/quant-data-sig-0f824afd14.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1500_RH
