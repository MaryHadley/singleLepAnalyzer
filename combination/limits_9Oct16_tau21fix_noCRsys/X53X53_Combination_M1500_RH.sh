#!/bin/sh 
cd /home/ssagir/CMSSW_7_3_0/src/
source /cvmfs/cms.cern.ch/cmsset_default.sh
cmsenv
cd -
cd limits_9Oct16_tau21fix_noCRsys/
/home/ssagir/CMSSW_7_3_0/src/theta/utils2/theta-auto.py /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1500_RH.py