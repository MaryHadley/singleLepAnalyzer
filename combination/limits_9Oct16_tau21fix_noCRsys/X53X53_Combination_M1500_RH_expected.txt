# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.069895   0.035485    0.14702  0.048258    0.10346
